src/primitives/felt.test.ts

```diff
@@ -1,20 +1,24 @@
 import { test, expect, describe } from 'bun:test';
 import { Felt } from './felt';
-import { OutOfRangeBigInt, PrimitiveError } from 'errors/primitives';
-import { Relocatable } from './relocatable';
 
 describe('Felt', () => {
   describe('constructor', () => {
-    test('should throw if initializing a felt with a negative inner', () => {
-      expect(() => new Felt(-10n)).toThrow(
-        new PrimitiveError(OutOfRangeBigInt)
+    test('should properly initialize a Felt with a negative bigint as parameter', () => {
```

Comment:
it looks like you want to "parametrize"  your test to have several cases covered, see "jest parameterized test" in google